index.js

```diff
@@ -1,32 +1,155 @@
-/**
- * (c) 2010-present DEMOS E-Partizipation GmbH.
- *
- * This file is part of the package @demos-europe/demosplan-ui,
- * for more information see the license file.
- *
- * All rights reserved
- */
 import { Tooltip, VPopover } from './directives/Tooltip/Tooltip'
 import { CleanHtml } from './directives/CleanHtml/CleanHtml'
-import DpButton from './components/DpButton/DpButton'
-import DpDetails from './components/DpDetails/DpDetails'
-import DpIcon from './components/DpIcon/DpIcon'
-import DpInput from './components/DpInput/DpInput'
-import DpLabel from './components/DpLabel/DpLabel'
-import DpLoading from './components/DpLoading/DpLoading'
 import prefixClass from './lib/prefixClass'
 import prefixClassMixin from './mixins/prefixClassMixin'
 
-export {
-  CleanHtml,
+import { exactlengthHint, maxlengthHint, minlengthHint } from './src/utils/'
+
+import {DpButton} from './src/components/'
+import {DpDetails} from './src/components/'
+import {DpIcon} from './src/components/'
+import {DpInput} from './src/components/'
+import {DpLabel} from './src/components/'
+import {DpLoading} from './src/components/'
```

Comment:
```suggestion
import {
    DpButton,
    DpDetails,
    DpIcon,
    DpInput,
    DpLabel,
    DpLoading
} from './src/components/'
```

Reply:
[64abf45](https://github.com/demos-europe/demosplan-ui/pull/13/commits/64abf452a7b1eca007906c680653a8916dbd8bc0) however not fully sorted alphabetically yet